sources/ClangSharp/Wrappers/CXCursorSet.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace ClangSharp
+{
+    public unsafe partial struct CXCursorSet
+    {
+        public CXCursorSet(IntPtr pointer)
+        {
+            Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+
+        public static implicit operator CXCursorSet(CXCursorSetImpl* value)
```

Comment:
Using implicit operators to convert from the native type to the wrapper type when it is an "exact" match.